extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -223,6 +226,29 @@ class AmpYoutube extends AMP.BaseElement {
       params['playsinline'] = '1';
     }
 
+    if ('loop' in params) {
+      // Loop is managed by the amp-youtube extension, prefer loop param instead
+      this.user().warn(
+        'AMP-YOUTUBE',
+        'Use loop attribute instead of the deprecated data-param-loop'
+      );
+    }
+
+    // In the case of a playlist, looping is delegated to the Youtube player
+    // instead of AMP manually looping the video through the Youtube API
+    const hasLoop =
```

Comment:
nit: hold the value `hasLoop` and `isPlaylist`  in instance variables and reuse in `layoutCallback` so the logic is not repeated.